### ISSUE TITLE
Feature to set checklist as per Schema of post

### ISSUE THREAD

**Issue by yogeshbeniwal** [publishpress/PublishPress-Checklists#285]
**Is your feature request related to a problem? Please describe.**
Post in WordPress are marked with Schema as Article/NewsArticle/Tech Article. Generally these types require different criteria as a quality guideline or pre-publishing checklist.

**Describe the solution you'd like**
Option to create checklist for post as per Schema type

**Comment by stevejburge** [comment]
@yogeshbeniwal thanks for the idea. There's not a standard way to set the schema in WordPress currently. Would this have to rely on another plugin?

**Comment by yogeshbeniwal** [comment]
Yeah, additional plugin is used to set Schema, Yoast SEO is one of them.

No sure about it if this has to be plugin specific. All these plugin add JSON-LD structured data to post.

**Comment by CodingSamurai** [comment]
Another vote for this. but we're using AISEO.  If you cover those two plugins that gets you something like 70% of the market for SEO plugins.
